nsv13/code/modules/overmap/ftl.dm

```diff
@@ -198,7 +198,7 @@
 					var/name = pick(GLOB.teleportlocs)
 					var/area/target = GLOB.teleportlocs[name]
 					var/turf/T = pick(get_area_turfs(target))
-					new /obj/effect/temp_visual/explosion_telegraph(T)
+					new /obj/effect/temp_visual/explosion_telegraph(T, damage_amount = ((world.time - structure_crit_init)*10/3))
```

Comment:
You may want to revise the scaling here. Right now it reaches maximum intensity after 6 seconds.

Reply:
Yup because I was calculating with minutes in my head, very cool 👍